TAC/Meetings/2022-04-21/CCC_TAC_Minutes-2022-04-21.md

```diff
@@ -0,0 +1,190 @@
+
+1. **[Kurt Taylor]** Investigate prior efforts to finalize Graduated lifecycle language, and if needed file an issue to finalize Graduated lifecycle language. (DT) summarized options for resolution: 1) Adopt new stages 2) talk about proposal and fix it 3) delete the proposed new terms and just use the the two currently adopted stages, sandbox and incubation
+
+2. **[Kurt Taylor]** Check if terminology document markdown format will work with LF Creative team and check on their timeline for completing the formatting for the whitepaper (Done)
+   * Markdown is fine, timeline is 1-3 weeks, depending on length and complexity)
+
+2. **[Kurt Taylor]** Code scanning via LFX Security - add all missing projects to security for scanning, get more details on instructions on installing github bot (Done) 
+    * Projects can set up the bot and repos they want scanned by following these instructions: https://community.lfx.dev/t/how-to-install-and-configure-bots-to-secure-your-projects/181
+
+
+### Completed Action Items
+
+
+### Project overviews
+
+
+### Interns
+* Dan Middleton (DM) - many new interns introduced in slack - needs clarity on where the interns are assigned
+* Need both Nick Vidal (NV) and DM here - keep on agenda for next meeting
+
+### Project website funding (Gramine)
+ * DT Gramine requested funding for a website, theme rebuild and managed infrastructure
+ * DT asking for approval - maybe need official policy with review of costs, may need higher budget, propose putting up for vote
+ * Jethro Beekman (JB) pointed out that this may be too expensive $150 for simple website, maybe go with another service not managed by LF and save money, can use the budget for another service
+ * DM - make it easy for the project, up to $150 as a budget
+ * DT - pointed out that the specific amount would be approved on a case-by-case basis
+ * DT reviewed actual website
+ * DT - would like to see a vote as Gramine is waiting for response
+ * DM - asked for context from Eric Voit (EV) as their mentor, EV reported that they have discussed design and proposal
+ * JB proposed website maintenence education (wordpress) by the LF for the same amount for upfront cost $500, for creation or education
+ * Lily Sturmann (LS)  asked to vote upfront and monthly seperately, DT said it wouldn't help Gramine, because it will be a signup as a package deal
+ * DT proposed up to $1800 year, $1000 standard policy for each project to use, vote on each project actual cost on a case-per-case basis as long as it falls into this budget
+ * Simon Johnson (SJ) asked for a more specific policy with guidelines for what the TAC would be looking for and what would be approved and what would not
+ * DT said that he agreed with SJ and that consistancey across the projects is a good thing, SJ said that some clarification on overall professionsal website design would be good
+ * DT called for a vote - slide 11 Proposal
+
+ **Approved** - Policy for new budget approved for website
```

Comment:
Instead of referencing Slide 11 Proposal could you just put the text here that was approved?